sdk/resourceconnector/Azure.ResourceManager.ResourceConnector/src/Generated/Models/Distro.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.ResourceConnector.Models
+{
+    /// <summary> Represents a supported Fabric/Infra. (AKSEdge etc...). </summary>
+    public readonly partial struct Distro : IEquatable<Distro>
```

Comment:
Avoid using single word which is too generic. Consider appending RP prefix or renaming it.
- use `prepend-rp-prefix` to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).
- use `rename-mapping` to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)

Reply:
That applies to other types like `SSHKey`

Reply:
> Not sure I understand the ask here? Is this to rename the Generated SDK model? That would not be possible right as this is derived from the Swagger. Is the ask here to update the `autorest.md` for naming convention? (Wondering if this should be left as to be uniform with other SDKs? )

@aditimalladi Yes. The comment is about naming conventions to follow [Azure .NET SDK guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html). Serialization logic will ensure the names in JSON will still be the ones defined in swagger.  You can take a look at the links I posted above.